coalib/coala.py

```diff
@@ -42,18 +42,39 @@ def main():
             return mode_json(args)
 
         if args.show_bears:
-            from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
-            show_bears(local_bears,
-                       global_bears,
-                       args.show_description or args.show_details,
-                       args.show_details,
-                       console_printer)
-
+            if args.filter_by:
+                from coalib.parsing.Filter import Filter
+                filter = args.filter_by[0]
+                filter_args = args.filter_by[1:]
```

Comment:
`or` operator is allowed only within a single filter.
It means we can write `filter-by C or python`but not`filter-by C or filter-by can_fix syntax`. Although I'm not sure later is that useful. It is mainly useful with`and`operators, which I'm a working on to implement.